Docs/rename and replace some sections (#90)
deeppavlov/dialog_flow_framework: docs/source/examples.rst

Examples
--------

> Work in progress... Examples will be available in this `repository <https://github.com/deeppavlov/dialog_flow_demo>`_.

Reviewer: They are already available though.
```suggestion
Work in progress... Examples are available in this `repository <https://github.com/deeppavlov/dialog_flow_demo>`_.
```
